templates/travel/components/TravelHero.js

```diff
@@ -0,0 +1,237 @@
+import TravelHeroSearch from '../components/TravelHeroSearch';
```

Comment:
Once accepted, should I merge all my changes into this master repo and refork to keep working? Or should we delete everything from here (the Travel template) and reimport it?

Reply:
I'd suggest you create PR against the amstart-2.0 branch